packages/appstore/src/modules/onboarding/onboarding.scss

```diff
@@ -3,76 +3,91 @@
     flex-direction: column;
     background-color: var(--text-less-prominent);
     height: inherit;
-}
-
-.onboarding-header {
-    display: flex;
-    margin-left: 1rem;
-    justify-content: space-between;
-
-    &__cross-icon {
-        margin: 3.5rem 3rem 0 0;
-        cursor: pointer;
-    }
-}
-
-.onboarding-body {
-    display: flex;
-    justify-content: center;
-    color: var(--general-main-1);
+    .onboarding-header {
+        display: flex;
+        margin-left: 1rem;
+        justify-content: space-between;
 
-    @include mobile {
-        overflow-y: scroll;
-        height: 42rem;
+        &__cross-icon {
+            margin: 3.5rem 3rem 0 0;
+            cursor: pointer;
+        }
     }
-}
 
-.onboarding-footer {
-    background-color: var(--general-main-1);
-    width: 100%;
-    height: 100%;
-
-    @include mobile {
-        position: fixed;
-        top: 75%;
-    }
+    .onboarding-body {
+        display: flex;
+        justify-content: center;
+        color: var(--general-main-1);
 
-    &__progress-bar {
-        margin-top: 2rem;
+        @include mobile {
+            overflow-y: scroll;
+            height: 42rem;
+        }
     }
 
-    &-header {
-        line-height: 3rem;
-    }
+    .onboarding-footer {
+        background-color: var(--general-main-1);
+        width: 100%;
+        height: 100%;
 
-    &-text {
-        line-height: 2rem;
-        margin-top: 0.8rem;
         @include mobile {
-            margin-top: 0;
+            position: fixed;
+            top: 75%;
         }
-    }
 
-    &-buttons {
-        display: flex;
-        justify-content: space-around;
-        margin: 2.4rem 0;
+        &__progress-bar {
+            margin-top: 2rem;
+        }
 
-        @include mobile {
-            width: 100%;
-            margin: 1.5rem 0;
-            overflow-y: auto;
-            &--full-size {
-                width: 80%;
-                height: 3.8rem;
-                margin-left: 11%;
+        &-header {
+            line-height: 3rem;
+        }
+
+        &-text {
+            line-height: 2rem;
+            margin-top: 0.8rem;
+            @include mobile {
+                margin-top: 0;
             }
         }
-        &--mobile {
+
+        &-buttons {
+            display: flex;
+            justify-content: space-around;
+            margin: 2.4rem 0;
+
             @include mobile {
-                height: 3.8rem;
-                width: 40%;
+                width: 100%;
+                margin: 1.5rem 0;
+                overflow-y: auto;
+                &--full-size {
+                    width: 80%;
+                    height: 3.8rem;
+                    margin-left: 11%;
```

Comment:
Not recommended at all to use percentage for alignments of the page and its content. @mahdiyeh-fs

Reply:
I've just moved the previous styles to the parent class, but will change this accordingly.

Reply:
@matin-binary we need lots of modifications in onboarding page, so this will handle with those since not related to my PR :pray: